scripts/release-post/index.mjs

```diff
@@ -25,6 +25,7 @@ import { writeFile } from 'node:fs/promises';
 import { resolve } from 'node:path';
 
 import handlebars from 'handlebars';
+import { format } from 'prettier';
```

Comment:
Does this read our prettier settings by default? `.prettierrc.json`

Reply:
Not yet, I'm currently using default options. 
Will import `.prettierrc.json` configs for custom options. Thanks 🙇🏻‍♂️